config/locales/sufia.en.yml

```diff
@@ -166,3 +168,5 @@ en:
       department:
         text: "Department"
         label_html: "Department"
+      open:
+         label_html: Open Access
```

Comment:
something's wrong here. we shouldn't need to add these here, they're already a part of Sufia.